src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -421,6 +421,12 @@ public enum CassandraRelevantProperties
     SAI_VECTOR_FLUSH_THRESHOLD_MAX_ROWS("cassandra.sai.vector_flush_threshold_max_rows", "-1"),
     // Use non-positive value to disable it. Period in millis to trigger a flush for SAI vector memtable index.
     SAI_VECTOR_FLUSH_PERIOD_IN_MILLIS("cassandra.sai.vector_flush_period_in_millis", "-1"),
+    // Use nvq when building graphs in compaction. Disabled by default for now. Enabling will reduce recall slightly
+    // while also reducing the storage footprint.
+    SAI_VECTOR_ENABLE_NVQ("cassandra.sai.vector.enable_nvq", "false"),
+    // NVQ number of subvectors. This isn't really expected to change much so we're only exposing
+    // it as a global variable in case it's needed.
+    SAI_VECTOR_NVQ_NUM_SUB_VECTORS("cassandra.sai.vector.nvq_num_sub_vectors", "2"),
```

Comment:
Some other settings use `cassandra.sai.vector_search.` prefix.
E.g. `cassandra.sai.vector_search.max_top_k`

However, those apply to the query processing part (search), and this here to index construction, so maybe that's fine. Wdyt?

Reply:
I agree, these aren't technically search configs. I also see that we have `cassandra.sai.non_vector_flush_threshold_max_rows` and `cassandra.sai.vector_flush_threshold_max_rows` so it would seem there isn't one standard. I tend to like the `.vector.` namespace effect, but it also doesn't technically indicate that the config is going to influence graph construction.